components/common/Card.tsx

```diff
@@ -130,6 +132,8 @@ const Card = (props: CardProps) => {
             $background={imageUrl}
             $imageAlign={imageAlign}
             data-testid="image-bg"
+            role="img"
+            aria-label={t('image-credit')}
```

Comment:
Maybe this should be `image. altText` (although it's not always filled, but then the responsibility is on content editor)?

Reply:
@terotik I added a condition to use ariaText for aria-label if it's provided. Otherwise, we use generic "image"

Reply:
Also, maybe we should make the aria-text field for images required in the Admin to avoid empty aria-labels for images?